### DISCUSSION TITLE
A small piece of the background was not removed

### DISCUSSION THREAD

**Discussion by Winky678** [danielgatis/rembg#702] [Q&A]
<img width="1190" alt="image" src="https://github.com/user-attachments/assets/ddf8c3ad-6cf1-49f9-bab1-283a56d55313" />
Hello, I want to know if there are any parameters that can change the background of this small area to the color I want?

**Comment by Winky678** [comment]
```
def change_background(input_image_path, output_image_path, background_color=(255, 255, 255)):
    try:
        with open(input_image_path, 'rb') as input_file:
            input_image = input_file.read()
        foreground = remove(input_image)
        foreground.save("xyq.png")
        foreground_image = Image.open(io.BytesIO(foreground)).convert("RGBA")

        background_image = Image.new("RGBA", foreground_image.size, background_color + (255,))

        combined_image = Image.alpha_composite(background_image, foreground_image)

        combined_image = combined_image.convert("RGB")
        combined_image.save(output_image_path)
        return True
    except Exception as e:
        print(f"Error : {e}")
        return False
```

**Comment by proremovebg** [comment]
I’ve seen this happen when an automatic background-removal tool leaves behind small patches around edges or detailed areas.

One thing that can help is processing the image with a background-removal tool that is designed to detect fine edges and small remaining background areas. You can try **[ProRemoveBG](https://proremovebg.com/)** 

It may be worth comparing the result with the current output, especially if the remaining background is around hair, fur, thin objects, or other detailed edges.
